cognite/client/data_classes/simulators/routines.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import TYPE_CHECKING, Any
+
+from typing_extensions import Self
+
+from cognite.client.data_classes._base import (
+    CogniteResourceList,
+    ExternalIDTransformerMixin,
+    IdTransformerMixin,
+    WriteableCogniteResource,
+    WriteableCogniteResourceList,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+
+class SimulatorRoutineCore(WriteableCogniteResource["SimulatorRoutineWrite"], ABC):
+    """
+    The simulator routine resource defines instructions on interacting with a simulator model. A simulator routine includes:
+
+    * Inputs (values set into the simulator model)
+    * Commands (actions to be performed by the simulator)
+    * Outputs (values read from the simulator model)
```

Comment:
does it make sense to mention them here, considering it only makes sense on the routine rev level?